lib/cc/analyzer/container.rb

```diff
@@ -116,7 +154,11 @@ def reap_running_container
       end
 
       def timeout
-        (ENV["CONTAINER_TIMEOUT_SECONDS"] || DEFAULT_TIMEOUT).to_i
+        ENV.fetch("CONTAINER_TIMEOUT_SECONDS", DEFAULT_TIMEOUT).to_i
```

Comment:
That snippet results in `""` in `irb`.

I'm pretty sure the only way you can get a `nil` in `ENV` is if the Ruby process explicitly sets it.